scripts/tests/test_json_config_expand_relpaths_script/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ function(add_json_config_expand_relpaths_test name)
     COMMAND ${NODEJS_EXECUTABLE} ${JSON_CONFIG_EXPAND_RELPATHS_SCRIPT} ${ARGN}
     )
   set(testname ${testname} PARENT_SCOPE)
+  set_property(TEST "${testname}" PROPERTY LABELS girder_devel)
```

Comment:
Exactly.  In case people forget to add a label to a test, it will be run in this group.